### PR TITLE
add information to html

### PR DESCRIPTION
**Pull request by MonaHer**
Added first information to my personal website

### CODE REVIEW

**Comment by CorneliusMoe** [comment]
Sieht gut aus 🙂

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <a href="#biography">Biography</a>
+  </nav>
+    </header>
+    <main>
+        
+        <section id="start">
+<figure>
+    <img src="https://placekitten.com/640/360"/>
+    <figcaption>
+        xxxxxx
```
Comment (AliaksandraNovikava) [inline index.html:24]: I would recommend to use "lorem ipsum" if you don't have any content yet 😉

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      </section>
+
+
+
+
+
+
+    </main>
+    <footer>
+
```
Comment (AliaksandraNovikava) [inline index.html:60]: How about adding a p-tag with "lorem ipsum" here? This way you can add some content to your empty footer-element. 
Later you can add e.g. links to "privacy policy" and "terms and conditions".

**Review by AliaksandraNovikava — APPROVED** [review]
LGTM 😉

File: index.html (hunk 1 above)
Comment (makitango) [reply index.html:24]: Agreed. xxx brings up ambitious context. :D

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+<section id="about-me">
+    <h2>About me</h2>
+    <article>
+      Hi there! Welcome to my personal website. Here is gathered the most
+      important information about me. If you want to work with me or get to
+      know me, please don't hesitate to get in contact.
+    </article>
+    <section id="contact">
+        <h2>Contact</h2>
+        <!--When clicking on these element, nothing happens! But I want to be redirected to the other websites. Why is it not working?-->
```
Comment (makitango) [inline index.html:36]: We were both ruthless on this approach, but let's remove those hints. ;)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        <li><a href="https://github.com/neuefische">GitHub</a></li>
+        <li><a href="https://de.linkedin.com/">LinkedIn</a></li>
+        <li><a href="https://codesandbox.io/">CodeSandbox</a></li>
+      </section>
+      <h2>Skillset</h2>
+        <!--This part could use some bulletpoint to be more structured-->
```
Comment (makitango) [inline index.html:42]: see line \#36

File: index.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+        <ul>
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+          <li>React</li>
+          <li>Painting rainbows</li>
+          <li>Party</li>
+        </ul>
+      </section>
+
```
Comment (makitango) [inline index.html:52]: Those empty lines could be removed

**Review by makitango — APPROVED** [review]

**Review by josch87 — COMMENTED** [review]
I go with the other comments.

**Pull request merged**